src/cli/pytest_commands/pytest_ini_files/pytest-execute-recover.ini

```diff
@@ -2,9 +2,6 @@
 console_output_style = count
 minversion = 7.0
 python_files = *.py
-markers =
-    slow
-    pre_alloc_modify
```

Comment:
Note, recover doesn't collect tests from `./tests` and this marker registration was unnecessary. It also doesn't load the `pytest_plugins.shared.execute_fill` plugin.